app/views/crops/_plantings.html.haml

```diff
@@ -6,7 +6,7 @@
   %ul
     - crop.plantings.take(3).each do |planting|
       %li
-        = link_to "#{planting.owner} planted #{planting.quantity} #{planting.planted_from}.", planting_path(planting)
+        = link_to display_planting(planting), planting_path(planting)
```

Comment:
This may be my personal preference, but I would probably refactor this to look like:
```
= link_to planting.to_sentence, planting_path(planting)
```